src/zui/ZUIOrganizeSidebar/index.tsx

```diff
@@ -47,10 +53,116 @@ import ZUIUserAvatar from 'zui/ZUIUserAvatar';
 import useFeature from 'utils/featureFlags/useFeature';
 import { AREAS, OFFICIALS } from 'utils/featureFlags';
 import oldTheme from 'theme';
+import useIsMobile from 'utils/hooks/useIsMobile';
+import { ZetkinUser } from 'utils/types/zetkin';
 
 const drawerWidth = 300;
 
-const ZUIOrganizeSidebar = (): JSX.Element => {
+const ZUIOrganizeMobileHeader = ({
+  closeMobileSidebar,
+  openMobileSidebar,
+  title,
+  user,
+  userMenuItems,
+}: {
+  closeMobileSidebar: () => void;
+  openMobileSidebar: () => void;
+  title?: string | ReactElement;
+  user: ZetkinUser | null;
+  userMenuItems: ZUIEllipsisMenuItem[];
+}) => {
+  const [mobileUserMenuAnchor, setMobileUserMenuAnchor] =
+    useState<HTMLButtonElement | null>(null);
+  const theme = useTheme();
+
+  return (
+    <Box
+      sx={{
+        alignItems: 'center',
+        backgroundColor: theme.palette.background.default,
+        borderBottom: `1px solid ${theme.palette.grey['300']}`,
+        display: 'flex',
+        flexDirection: 'row',
+        height: '3.5rem',
+        justifyContent: 'space-between',
+        left: 0,
+        padding: '0 5px',
+        pointerEvents: 'all',
+        position: 'sticky',
+        top: 0,
+        width: '100vw',
+        zIndex: 1,
+      }}
+    >
+      <Box
+        sx={{
+          alignItems: 'center',
+          display: 'flex',
+          flexDirection: 'row',
+        }}
+      >
+        <Button
+          onClick={(e) => {
+            openMobileSidebar();
+            e.stopPropagation();
+          }}
+        >
+          <MoreVert />
+        </Button>
+        <Typography fontSize={'16px'} variant={'h1'}>
+          {title}
+        </Typography>
+      </Box>
+      {user && (
+        <>
+          <Button
+            onClick={(event) => setMobileUserMenuAnchor(event.currentTarget)}
+          >
+            <ZUIUserAvatar personId={user.id} size={'sm'} />
+          </Button>
+          <Menu
+            anchorEl={mobileUserMenuAnchor}
+            onClose={() => setMobileUserMenuAnchor(null)}
+            open={Boolean(mobileUserMenuAnchor)}
+          >
+            {userMenuItems.map((item, index) => {
```

Comment:
I can't log out when pressing the log out button up here in the header. It only works at the bottom of the drawer.